src/modules/completeStatus.js

```diff
@@ -16,11 +14,10 @@ export const markAsCompleted = (todo, listItem) => {
     setData();
   }
 };
-
+// finally got it to work
 export const completed = (todo) => {
   const completedTasks = todo.filter((task) => task.completed === false);
   overwriteIndex(completedTasks);
   overWriteLiId();
   localStorage.setItem('toDo', JSON.stringify(completedTasks));
```

Comment:
- [ ] Kindly fix all linters errors instead of disabling them in the `.eslintrc.json` config. For the default `eslintrc.json` file you can follow this link [(linters config)](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)